x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.ts

```diff
@@ -537,6 +537,7 @@ const endpointsTotal = async (http: HttpStart): Promise<number> => {
       })
     ).total;
   } catch (error) {
+    // TODO
```

Comment:
could you add an explanation next to `TODO`? 
```
// TODO: some explanation here
```

Reply:
done 5f28720ea050d8a1173b590be0da4705b45170b5